morphia/src/main/java/org/mongodb/morphia/query/QueryImpl.java

```diff
@@ -279,14 +310,17 @@ protected BasicDBObject copy(final BasicDBObject dbObject) {
     }
 
     @Override
+    @Deprecated
     public Query<T> disableCursorTimeout() {
-        noTimeout = true;
+        getOptions().noCursorTimeout(true);
         return this;
     }
 
     @Override
+    @Deprecated
     public Query<T> disableSnapshotMode() {
```

Comment:
the getBlahDBObject() methods were there so that I could migrate to `Document` based code at some point.  Instead, i've just removed them from the public API altogether so I can change them in the future without breaking user code.